custom/icds/messaging/indicators.py

```diff
@@ -35,6 +36,36 @@
 
 DEFAULT_LANGUAGE = 'hin'
 
+REPORT_IDS = [
+    HOME_VISIT_REPORT_ID,
+    THR_REPORT_ID,
+    CHILDREN_WEIGHED_REPORT_ID,
+    DAYS_AWC_OPEN_REPORT_ID,
+]
+
+
+@quickcache(['domain'], timeout=4 * 60 * 60, memoize_timeout=4 * 60 * 60)
+def get_report_configs(domain):
+    app = get_app(domain, SUPERVISOR_APP_ID)
```

Comment:
Yeah. I agree. I should have used the latest  starred build for this. although I imagine there won't be too much app dev happening on NIC